utils/svg2png/svg2png.cpp

```diff
@@ -71,17 +73,34 @@ struct main_marker_visitor
         double opacity = 1;
         double w, h;
         std::tie(w, h) = marker.dimensions();
+
+
+        double svg_width = w * scale_factor_ + 0.5;
+        double svg_height = h * scale_factor_ + 0.5;
```

Comment:
adding 0.5 here seems strange, see below...

Reply:
The idea is to have output image in pixels with dimensions equal or slightly large then size reported by SVG which is floating point numbers.

Reply:
I should've put emphasis on *here*. Was wondering primarily why 0.5 was added *here* to the floating-point scaled width, rather than only inside the rounding expression. But now with your explanation, it sounds like it was meant to be rounding up, instead of rounding to nearest:
```c++
double svg_width = w * scale_factor_;
int output_width = static_cast<int>(std::ceil(svg_width));
```

Reply:
My comment disappeared, doh.. retyping:  Agree with moving up/down 0.5 scaling inside rounding expression.
std::ceil is not what others (checked on Ubuntu) are doing see details at #4116